src/components/app/App.js

```diff
@@ -219,7 +223,22 @@ render() {
         }
     ];
 
+
+
     return (
-        <UserDetails />
-    )
+    users.map((el,index) => {
+        return (
+            <div>
```

Comment:
Why do you create all the new structure, if you already created User component. The idea is that you take array and map it to User components:

`users.map((user, index)=>(<UserDetails ...props />))`

Reply:
Тут div не обов'язково вставляти. А от `key` до кожного елемента треба додати. Якщо у тебе є ESLint, він повинен був би показати помилку. 

Just in case you are wondering, why we need keys to elements, here is a nice explanation:
https://medium.com/@adhithiravi/why-do-i-need-keys-in-react-lists-dbb522188bbb